examples/TestAppUwp/MainPage.xaml.cs

```diff
@@ -262,47 +75,54 @@ public static string GetDeviceName()
+            //        };
+            //        AutomationProperties.SetName(item, cat.Name);
+            //        navigationView.MenuItems.Add(item);
+            //    }
+            //    else if (catBase is Header headerCat)
+            //    {
+            //        var item = new NavigationViewItemHeader()
+            //        {
+            //            Name = headerCat.Name,
+            //            DataContext = catBase
+            //        };
+            //        AutomationProperties.SetName(item, headerCat.Name);
+            //        navigationView.MenuItems.Add(item);
+            //    }
+            //    else if (catBase is Separator sepCat)
+            //    {
+            //        navigationView.MenuItems.Add(new NavigationViewItemSeparator());
+            //    }
+            //}
 
-            // This will be enabled once the signaling is initialized and ready to send data
-            createOfferButton.IsEnabled = false;
+            // Open the "Signaling" page by default
+            //navigationView.MenuItemsSource = Categories;
+            rootFrame.Navigate((Categories[0] as Category).PageType);
+            //((NavigationViewItem)(navigationView.MenuItems[0])).IsSelected = true; // doesn't work...
 
             // Those are called during InitializeComponent() but before the controls are initialized (!).
             // Force-call again to actually initialize the panels correctly.
-            PreferredAudioCodecChecked(null, null);
-            PreferredVideoCodecChecked(null, null);
-
-            RestoreParams();
-
-            dssSignaler.OnMessage += DssSignaler_OnMessage;
-            dssSignaler.OnFailure += DssSignaler_OnFailure;
-            dssSignaler.OnPollingDone += DssSignaler_OnPollingDone;
-
-            localVideoStatsTimer.Tick += OnLocalVideoStatsTimerTicked;
-            remoteVideoStatsTimer.Tick += OnRemoteVideoStatsTimerTicked;
-
-            _peerConnection = new PeerConnection();
-            _peerConnection.Connected += OnPeerConnected;
-            _peerConnection.DataChannelAdded += OnDataChannelAdded;
-            _peerConnection.DataChannelRemoved += OnDataChannelRemoved;
-            _peerConnection.LocalSdpReadytoSend += OnLocalSdpReadyToSend;
-            _peerConnection.IceCandidateReadytoSend += OnIceCandidateReadyToSend;
-            _peerConnection.IceStateChanged += OnIceStateChanged;
-            _peerConnection.IceGatheringStateChanged += OnIceGatheringStateChanged;
-            _peerConnection.RenegotiationNeeded += OnPeerRenegotiationNeeded;
-            _peerConnection.AudioTrackAdded += Peer_RemoteAudioTrackAdded;
-            _peerConnection.AudioTrackRemoved += Peer_RemoteAudioTrackRemoved;
-            _peerConnection.VideoTrackAdded += Peer_RemoteVideoTrackAdded;
-            _peerConnection.VideoTrackRemoved += Peer_RemoteVideoTrackRemoved;
+            //PreferredAudioCodecChecked(null, null);
+            //PreferredVideoCodecChecked(null, null);
```

Comment:
Delete?

Reply:
Let's skip all stale/unused code comments, there are 2 other changes coming after this one, and some others needed to polish the app. I just want to merge multitrack ASAP in master.

Reply:
It would be nice to add TODOs where there's remaining work, but I'll resolve if these are going to be fixed shortly.